pkgs/top-level/beam-packages.nix

```diff
@@ -0,0 +1,95 @@
+{ pkgs, stdenv, callPackage }:
+
+rec {
+  lib = import ../development/beam-modules/lib.nix { inherit pkgs; };
+
+  inherit (pkgs) darwin wxGTK30;
```

Comment:
Moved these to arguments of this file (i.e. lambda defined at top level of this file)